processing/src/main/java/org/apache/druid/segment/incremental/IncrementalIndexStorageAdapter.java

```diff
@@ -295,6 +300,16 @@ public Metadata getMetadata()
     return index.getMetadata();
   }
 
+  @Override
+  public List<String> getSortOrder()
+  {
+    if (index.timePosition == 0) {
+      return Metadata.SORTED_BY_TIME_ONLY;
+    } else {
+      return Collections.emptyList();
```

Comment:
i suppose depending on the type of facts holder we could actually report something here (rollup should be ordered i think?)

Reply:
I think that makes sense. I added a comment about this being a possible change in the future.